core/src/main/scala/com/softwaremill/kmq/redelivery/streams/CommitMarkerStream.scala

```diff
@@ -0,0 +1,69 @@
+package com.softwaremill.kmq.redelivery.streams
+
+import akka.Done
+import akka.actor.ActorSystem
+import akka.kafka.ConsumerMessage.CommittableMessage
+import akka.kafka.scaladsl.Consumer.DrainingControl
+import akka.kafka.scaladsl.{Committer, Consumer}
+import akka.kafka.{CommitterSettings, ConsumerSettings, Subscriptions}
+import akka.stream.scaladsl.{Flow, Keep, Sink}
+import com.softwaremill.kmq.redelivery.Offset
+import com.softwaremill.kmq.{EndMarker, MarkerKey, MarkerValue, StartMarker}
+
+import scala.concurrent.Future
+
+class CommitMarkerStream(markerConsumerSettings: ConsumerSettings[MarkerKey, MarkerValue],
```

Comment:
[naming]: this class doesn't define a stream, but a sink. So maybe call it a `Sink` and the creation method simply `apply` or `create`?